public/pages/main/Main.tsx

```diff
@@ -122,22 +147,29 @@ export function Main(props: MainProps) {
                 />
                 <Route
                   path={APP_PATH.OVERVIEW}
-                  render={() => (
+                  render={(props: RouteComponentProps) => (
                     <AnomalyDetectionOverview
-                      isLoadingDetectors={isLoadingDetectors}
```

Comment:
Have you confirmed the impact of removing this? Do we still have proper loading state when loading the page and creating sample detectors? Adding details in the PR for this may be helpful.

Reply:
I'm not deleting this isLoadingDetectors line, just passing it as a prop from here doesn't make sense. So I moved it inside AnomalyDetectionOverview page and also added another selector from open search reducer as we're also loading sample data/indices - https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/f166bd2463d5d314a6138ee082c8a72c92e26231/public/pages/Overview/containers/AnomalyDetectionOverview.tsx#L79-L80